index.html

```diff
@@ -513,29 +513,140 @@ <h3>Assert Claim</h3>
+        </dd>
+        <dt>Needs</dt>
+        <dd>
+          <uref>F.2</uref>, <uref>C.1</uref>, <uref>E.2</uref>, <uref>R.1
+          </uref>,
+          <uref>F.5</uref>, <uref>H.2</uref>, <uref>C.6</uref>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h3>Validate Claim</h3>
+      <dl class="dl-horizontal">
+        <dt>Requirement</dt>
+        <dd>
+          It MUST be possible for a <a>verifier</a> to validate that the a
+          given claim is appropriate for particular use. That is, given the
+          credential is authentic and current, the verifier must then
+          interpret the claims in the context of their own 'business rules'
+          about which parties are acceptable authorities and which
+          cryptographic mechanisms are acceptable for different situations.
+          <ol>
+            <li>Is the issuer someone we know?</li>
+            <li>Are the claims made by this issuer appropriate for that
+            particular issuer's authority?</li>
+            <li>Is the holder's presentation an appropriate use of the
+            credential?</li>
+          </ol>
+          The first and second question will be answered either because the
+          verifier already knows the public identifiers for the issuers it
+          recognizes for specific claims, or there is a discovery mechanism
+          whereby verifiers can find appropriate issuers given their
+          business requirements. 
+          
+          The business rules may allow the use of certain credentials for
+          specific situations even when they are expired. For example a
+          suspended professional license may be accepted by an employer as
+          evidence of past experience without treating it as a current
+          qualification for roles that require that license. For this
+          reason, currency (as part of verification) only deals with the
+          status of the credential as maintained by the Issuer. Timeliness
+          is the domain of validation.
+
+          It's important to note that the bounds of authority for a given
+          issuer are specific claim types made by them and relied upon by
+          others. While the DMV might be an acceptable authority for an
+          individual's driving privilege, it should probably not be treated
+          as authoritative for an individual's hair color or current
+          address, even though DMVs regularly include such claims in
+          today's driver's licenses.
+
+          Unlike verification, validation cannot be guaranteed to be
+          automatable. Some use cases require human review before accepting
+          a particular credential for a particular use. For example, a
+          digital proof of age system might need a trusted human agent to
+          visually compare the holder/presenter in real-space to a
+          reference photo securely referenced in the VC, before accepting a
+          credential as valid.
```

Comment:
```suggestion
          Unlike verification, it may not always be possible to
          automate validation. Some use cases require human review before accepting
          a particular credential for a particular use. For example, a
          digital proof of age system might need a trusted human agent to
          visually compare the holder/presenter in real-space to a
          reference photo securely referenced in the VC, before accepting a
          credential as valid.
```